Application/EdFi.Ods.Api/ScheduledJobs/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -8,11 +8,13 @@
 using log4net;
 using Microsoft.Extensions.DependencyInjection;
 using Quartz;
+using Quartz.AspNetCore;
 
 namespace EdFi.Ods.Api.ScheduledJobs.Extensions
 {
     public static class ServiceCollectionExtensions
     {
+        [System.Obsolete]
```

Comment:
Why is this marked as `obsolete`? I do not see any reason to change that during this .NET 8 upgrade.